### ISSUE TITLE
Pyxel install doesn't work on macbook

### ISSUE THREAD

**Issue by kdrangej** [kitao/pyxel#57]
Can someone help me resolve this issue? 

I followed the install instructions for mac but am unable to locate module pyxel. 

If I write `pyxel` the editor opens in a new window "Hello, Pyxel Editor!" 

```
this:pyxel_examples dev$ python3 01_hello_pyxel.py 
Traceback (most recent call last):
  File "01_hello_pyxel.py", line 1, in <module>
    import pyxel
ModuleNotFoundError: No module named 'pyxel'
```

**Comment by timbledum** [comment]
It sounds like you may have installed in a different version of python? Did you use `pip3` to install pyxel rather than `pip`?

Try typing `which pyxel` to see under which version pyxel has been installed. 

Otherwise, try installing in a new [environment](https://docs.python.org/3/tutorial/venv.html).

**Comment by kdrangej** [comment]
As I wrote above, I followed the instructions. 

This is what I get when I ask here pyxel has been installed
```
this:pyxel_examples dev$ which pyxel
/usr/local/bin/pyxel
```

**Comment by timbledum** [comment]
Yes – as I suspected it looks like pyxel is installed under your python 2 instance rather than python 3. For context, this is what I get when I run `which pyxel`.

    this:~ timbledum$ which pyxel
    /Library/Frameworks/Python.framework/Versions/3.7/bin/pyxel

Try running `pip3 install pyxel` again, and if that doesn't work try `python3 -m pip install pyxel`.
